lib/ui/settings/backup_section_widget.dart

```diff
@@ -103,7 +103,7 @@ class BackupSectionWidgetState extends State<BackupSectionWidget> {
             mainAxisAlignment: MainAxisAlignment.spaceBetween,
             children: [
               Text(
-                "Wakelock",
+                "Keep device awake",
```

Comment:
Alternative versions:

Disable screen lock

Disable automatic screen lock when ente is running?

This will ensure faster uploads by ensuring your device does not sleep when uploads are in progress.

Automatic screen lock disabled
Automatic screen lock enabled

Reply:
Versions 2:

Disable auto lock

Disable automatic screen lock when ente is running?

This will ensure faster uploads by ensuring your device does not sleep when uploads are in progress.

(no toast)